onyo/lib/assets.py

```diff
@@ -3,7 +3,7 @@
 
 import re
 from pathlib import Path
-from typing import Dict, Union, Iterable, Set, Generator, Optional
+from typing import Dict, Generator, Iterable, Optional, Set
```

Comment:
Thanks for sorting! :D

Reply:
Haha. Yeah. Most were sorted, and I just did the rest while I was there anyways.